radix-engine-common/src/math/decimal.rs

```diff
@@ -62,10 +64,11 @@ impl Decimal {
 
     pub const ZERO: Self = Self(I192::ZERO);
 
+    pub const ONE_HUNDREDTH: Self = Self(I192::from_digits([10_u64.pow(Decimal::SCALE - 2), 0, 0]));
```

Comment:
Can we test all these constants by converting it to string? I saw you've got a few but wanted to make sure we cover them all.

Reply:
Done